File: go-danmu/model/Message.go
package model

import "gorm.io/gorm"

type UserMessage struct {
	gorm.Model
	Uid     uint   `gorm:"not null;"` //用户ID
	Fid     uint   `gorm:"not null;"` //关联ID
	FromId  uint   `gorm:"not null;"` // 发送者
	ToId    uint   `gorm:"not null;"` // 接受者
	Content string `gorm:"size:255;"`
	Status  bool   `gorm:"default:false"` //已读状态
}

File: danmu-mobile/src/types/message.ts
export interface messageType {
    fid: number,
    content: string,
}

export interface messageListType {
    uid: number,
    name: string,
    avatar: string,
    created_at: string
}

export interface messageDetailsType {
    fid: number,
    from_id: number,
    content: string,
    created_at: string
}